simulation_manager/run_job.py

```diff
@@ -14,7 +14,7 @@
 
 # ============================================= JOB CONFIGURATION CONSTANTS ===========================================
 NUM_TRIALS = 1
-TIMEOUT = 300  # [s] kill a trial that takes longer than this amount
+TIMEOUT = 900  # [s] kill a trial that takes longer than this amount
```

Comment:
Is controller computation taking that much longer than dynamics? Python is slow, but this is a huge increase

Reply:
no, this was because I was running really long sims.